community/glide/APKBUILD

```diff
@@ -26,9 +26,15 @@ build() {
 	go build -o glide -ldflags "-X main.version=$pkgver" glide.go || return 1
```

Comment:
Commit (13e67c291ac92df3239e6bcd7dc76eeae96e5d2e) by @rgdoliveira upgraded Glide to 0.13.1.

So this PR could be updated to add the check.

` || return 1` should be removed.